payjoin/src/persist/mod.rs

```diff
@@ -0,0 +1,59 @@
+/// Types that can generate their own keys for persistent storage
+pub trait Value: serde::Serialize + serde::de::DeserializeOwned + Sized + Clone {
+    type Key: AsRef<[u8]> + Clone;
+
+    /// Unique identifier for this persisted value
+    fn key(&self) -> Self::Key;
+}
```

Comment:
The commit messages do not match the contents since `persist::Value` was introduced, not `Persistable`.

Traits tend to be named nouns or verbs (e.g. Read, Iterator, Clone)  rather than adjectives which is why I went with this name.

Reply:
Fixed